tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "moduleResolution": "Bundler",
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "noEmit": true
+    "noEmit": true,
+    "skipLibCheck": true
```

Comment:
Needed to add this because it was throwing a build error on build due to an issue with the `markdown-it` module. Adding this tells it not to get cranky if some of the node modules aren't easily compatible with ESM

Error was:

```
../../node_modules/@types/markdown-it/dist/index.cjs.d.ts:981:23 - error TS2709: Cannot use namespace 'LinkifyIt' as a type
```

Reply:
Removed in 167f11cab74d12c55501e35fa3be9ff6664c95e2